### PR TITLE
Improve capabilities

### PR DESCRIPTION
**Pull request by mucsi96**

### CODE REVIEW

File: docs/capabilities.md [hunk 1]
```diff
@@ -2,16 +2,54 @@
 
 ## Capabilities
 
+Before we can send any command to the browser we drive we need to create a [WebDriver](https://www.w3.org/TR/webdriver) session.
+This should be always the first step of interaction using the protocol.
+To create a new session we need to provide some requirements like `browserName`, `browserVersion`, `platformName`.
+These requirements are called [`Capabilities`](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities).
+This is espetially important when we interact not directly with a single browser but with a grid of browsers.
```
Comment (spnraju) [inline docs/capabilities.md]: especially misspelt.

File: docs/capabilities.md [hunk 2]
```diff
@@ -2,16 +2,54 @@
...
+Using `alwaysMatch` property we can provide some strict requirements.
+If the WebDriver sever can`t provide the required features the session creation will fail.
```
Comment (spnraju) [inline docs/capabilities.md]: Should that be single quote? the text is highlighted because of back ticks.

File: docs/capabilities.md [hunk 3]
```diff
@@ -2,16 +2,54 @@
...
+
+- [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities)
+
 ## BrowserCapability
 
 **PROPERTIES**
 
-- `browserName`: string
+- `browserName?`: string
+- `browserVersion?`: string
```
Comment (spnraju) [inline docs/capabilities.md]: Will these props be retained when we generate the docs again? Maybe when we add more apis are something.

**Review by spnraju — APPROVED** [review]

**Comment by mucsi96** [comment]
@spnraju you are awesome! Thanks for review and fixes 💯

**Pull request merged**